packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -60,7 +66,7 @@ Do not use actionable data lists when actions are restricted to a single row or
 ### Expandable data list
 The expandable list adds an expansion panel to every row to reveal more details about the item.
 
-![expandable data list](./img/expandable-list.png)
+<img src="./img/expandable-list.png"  alt="Expandable data list"  width="1174"/> 
```

Comment:
<img width="772" alt="Screen Shot 2020-06-22 at 1 48 52 PM" src="https://user-images.githubusercontent.com/35148959/85324303-37571b80-b48f-11ea-91a8-d9bd3d383873.png">

the "2nd" marker should be on the expansion - looks like it's on the toggle.